src/main/webapp/resources/js/pages/projects/samples/linker/Linker.jsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { render } from "react-dom";
+import { Button, Modal, Typography } from "antd";
+import { CodeOutlined } from "@ant-design/icons";
+import { grey2, grey4, grey9 } from "../../../../styles/colors";
+import { getNGSLinkerCode } from "../../../../apis/linker/linker";
+import { SPACE_SM } from "../../../../styles/spacing";
+
+const { Paragraph, Text } = Typography;
+
+/**
+ * React component to display the ngs-linker command to a user based
+ * on the currently selected samples.
+ * @returns {*}
+ * @constructor
+ */
+function Linker() {
+  function handleSampleIds(e) {
+    // Post data to get linker command.
+    const { detail } = e;
+    getNGSLinkerCode(detail).then(({ data }) => {
+      Modal.success({
+        className: "t-linker-modal",
+        width: 500,
+        title: i18n("Linker.title"),
+        content: (
+          <>
+            <Paragraph>{i18n("Linker.details")}</Paragraph>
+            <Text type="secondary">
+              <span
+                dangerouslySetInnerHTML={{ __html: i18n("Linker.note") }}
+              />
+            </Text>
+            <Paragraph
+              style={{
+                fontFamily: "monospace",
+                fontSize: 14,
+                marginTop: SPACE_SM,
+                padding: 2,
+                border: `1px solid ${grey4}`,
```

Comment:
We created a border const for this did we not?

Reply:
That is in your branch, I will look through it and maybe copy it.

Reply:
Not there, made a new one in 63e7cae5d2c7f6d864eb57e8f64057390df38a7f.

Reply:
Awesome thanks